micronaut-camunda-bpm-feature/src/test/kotlin/info/novatec/micronaut/camunda/bpm/feature/test/EventingPublisherPluginTest.kt

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package info.novatec.micronaut.camunda.bpm.feature.test
+
+import info.novatec.micronaut.camunda.bpm.feature.Configuration
+import info.novatec.micronaut.camunda.bpm.feature.MnProcessEngineConfiguration
+import info.novatec.micronaut.camunda.bpm.feature.eventing.EventingPublisherPlugin
+import info.novatec.micronaut.camunda.bpm.feature.eventing.ExecutionEvent
+import io.micronaut.context.event.ApplicationEventListener
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest
+import io.micronaut.test.support.TestPropertyProvider
+import org.camunda.bpm.engine.ProcessEngine
+import org.camunda.bpm.engine.RepositoryService
+import org.camunda.bpm.engine.RuntimeService
+import org.camunda.bpm.model.bpmn.Bpmn
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import javax.inject.Inject
+import javax.inject.Singleton
+
+
+@MicronautTest
+class EventingPublisherPluginTest {
+
+    @MicronautTest
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    inner class EventingTestPluginNotLoaded : TestPropertyProvider {
+
+        override fun getProperties(): MutableMap<String, String> {
+            return mutableMapOf()
+        }
+
+        @Inject
+        lateinit var processEngineConfiguration: MnProcessEngineConfiguration
+
+        @Inject
+        lateinit var processEngine: ProcessEngine
```

Comment:
* variable is never used, also applies to "configuration"
* also applies to other test methods